src/dp_vnf.c

```diff
@@ -189,10 +191,12 @@ bool dp_vnf_lbprefix_exists(uint16_t port_id, uint32_t vni, struct dp_ip_address
 		.port_id = port_id,
 		.vni = vni,
 		.type = DP_VNF_TYPE_LB_ALIAS_PFX,
-		.alias_pfx.length = prefix_len,
-		.alias_pfx.ol = *prefix_ip,
 	};
 
+	memset(&vnf.alias_pfx, 0, sizeof(vnf.alias_pfx));
+	vnf.alias_pfx.ol = *prefix_ip;
+	vnf.alias_pfx.length = prefix_len;
```

Comment:
Wait you are telling me that the old code was not setting this to zero? Because then the C standard is either wrong or I am reading it wrongly and having this problem in the whole codebase.
The old code should behave just like a global variable initialization to zero.

Reply:
The tests from dpservice passed.
The tests from dpservice-go passed.